### DISCUSSION TITLE
v13 end of life

### DISCUSSION THREAD

**Discussion by Romejanic** [discordjs/discord.js#9291] [Q&A] [answered]
Is there any planned EOL/support deadline for v13?

I'm just updating the documentation of my own package and I can't see any information about this.

**Comment by SuperchupuDev** [comment]
All we know is that it's not dead yet, it looks like it's in long term support for now based on the npm tags
![image](https://user-images.githubusercontent.com/53496941/228270360-3c3a9e2e-872c-4d98-9adb-bf5dbf44b347.png)

![image](https://user-images.githubusercontent.com/53496941/228270538-fd9aa2d5-2be8-41c7-b36a-6473a0e7fd08.png)

**Comment by megatank58** [comment, marked as answer]
The general eta had been when discord drops support for the api used by v13.

**Reply by Romejanic** [reply]
That makes sense, that's probably quite a while away I'm guessing ( v13 is on version 9 right?)

**Reply by kyranet** [reply]
Yes, it is. And in fact the Discord client also uses v9.
